src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.scss

```diff
@@ -2,38 +2,28 @@
 
 @import '/src/materialColorScheme.scss';
 
-// Header style - Expanded - Workflow
+// // Expansion panel background - Workflow
 mat-expansion-panel-header,
 mat-expansion-panel-header[aria-expanded='true']:hover,
 mat-expansion-panel-header[aria-expanded='true']:focus,
 :host ::ng-deep mat-expansion-panel .mat-expansion-panel-content {
   background-color: $workflow-background-color;
 }
 
-// Expansion body - Workflow
-:host ::ng-deep mat-expansion-panel.gitHubAppTool .mat-expansion-panel-content {
-  background-color: $workflow-background-color;
-}
-
 // Selected workflow style
 .selectedWorkflow {
   background-color: $workflow-selection-color;
 }
 
-// Header style - Expanded - GitHub App Tool
+// // Expansion panel background - GitHub App Tool
 mat-expansion-panel-header.gitHubAppTool,
 mat-expansion-panel-header.gitHubAppTool[aria-expanded='true']:hover,
 mat-expansion-panel-header.gitHubAppTool[aria-expanded='true']:focus,
 :host ::ng-deep mat-expansion-panel.gitHubAppTool .mat-expansion-panel-content {
   background-color: $tool-background-color;
 }
 
-// Expansion body - GitHub App Tool
-:host ::ng-deep mat-expansion-panel.gitHubAppTool .mat-expansion-panel-content {
-  background-color: $tool-background-color;
-}
-
 // Selected GitHub App Tool style
-.selectedGitHubAppTool {
+.selectedTool {
```

Comment:
Similar to my last comment, this is duplicated from mytools, so you could import the mytools styles to this component and get rid of this